client/components/SongDisplay.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SongMain from './SongMain.jsx';
+import SongArt from './SongArt.jsx';
+import SongExtras from './SongExtras.jsx';
+
+const SongDisplay = ({
+  song, playing, handleAlbumClick, handlePlayClick, handleInfoClick,
+}) => (
+  <div id="song-display">
+    <SongMain
+      song={song}
+      playing={playing}
+      handlePlayClick={handlePlayClick}
+      handleInfoClick={handleInfoClick}
+      id="song-display-main"
+    />
+    <SongExtras
+      song={song}
+      handleInfoClick={handleInfoClick}
+      id="song-display-extras"
```

Comment:
stylistic choice of having the id prop listed first or last but try to keep it consistent throughout

Reply:
Good call I'll make that change to have id come first!